scripts/README.md

```diff
@@ -0,0 +1,59 @@
+# FLARE Obfuscated String Solver
+
+## Scripts
+FLOSS currently supports converting its output into scripts for the following tools:
+1. [IDA pro](render-ida-import-script.py)
+2. [Binary Ninja](render-binja-import-script.py)
+3. [Ghidra](render-ghidra-import-script.py)
+4. [Radare2](render-r2-import-script.py)
+5. [X64dbg](render-x64dbg-database.py)
+  
+Additionally, there is another [plugin for IDA](idaplugin.py) to allow FLOSS to automatically
+extract obfuscated strings and apply them to the currently loaded module in IDA.
+
+# Installation
+These scripts can be downloaded from the FLOSS [GitHub](https://github.com/mandiant/flare-floss) repository
+alongside the source, which is required for the scripts to run.
+To install FLOSS as source, see the documentation [here](../doc/installation.md).
+
+
+# Usage
+#### To convert FLOSS output into scripts for tools:
+
+- Run FLOSS on the desired executable with the -j or --json argument to emit a JSON result
+and redirect it to a JSON file.
+
+
+    $ floss -j suspicious.exe > floss_results.json
+
+For Binary Ninja, IDA, Ghidra or Radare2,
```

Comment:
I think listing one example each is fine here and reduces potential maintenance.

Reply:
Thank you, I was wondering a bit about that. I changed it to only have one example. Also sorry for merging all commits separately, I noticed the batch option just a little bit too late.

Reply:
No worries, we can squash and merge here.